elog/scripts/LogBookPost.py

```diff
@@ -74,16 +75,20 @@
         params["msg"] = args.message
 
     if args.experiment:
-        elog = elog.ELog(logbooks={"experiment": args.experiment},
-                         user=args.user,
-                         pw=args.password,
-                         base_url=args.webserviceurl)
-        elog.post(**params)
+        elogger = elog.ELog(logbooks={"experiment": args.experiment},
+                            user=args.user,
+                            pw=args.password,
+                            base_url=args.webserviceurl)
+        elogger.post(**params)
     else:
-        elog = elog.HutchELog(args.instrument,
-                              station=args.station,
-                              user=args.user,
-                              pw=args.password,
-                              base_url=args.webserviceurl)
+        elogger = elog.HutchELog(args.instrument,
+                                 station=args.station,
+                                 user=args.user,
+                                 pw=args.password,
+                                 base_url=args.webserviceurl)
         params.update({"experiment": True, "facility": False})
-        elog.post(**params)
+        elogger.post(**params)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
(Picking a random line to comment)

Probably should add `elog.tests.__init__` to make it a proper submodule